<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PronList</class>
 <widget class="QDialog" name="PronList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Pronoun Conjugation List</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/machinetrans/resources/logo.png</normaloff>../../../../usr/share/machinetrans/resources/logo.png</iconset>
  </property>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>780</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Russian Pronoun List</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBttn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBttn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QListWidget" name="pronList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>780</width>
     <height>460</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addallBttn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Add All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpBttn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="singleBttn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Singles</string>
   </property>
  </widget>
  <widget class="QPushButton" name="groupBttn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Groups</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>helpBttn</tabstop>
  <tabstop>pronList</tabstop>
  <tabstop>cancelBttn</tabstop>
  <tabstop>singleBttn</tabstop>
  <tabstop>groupBttn</tabstop>
  <tabstop>addallBttn</tabstop>
  <tabstop>acceptBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
